public/locales/de/translation.json

```
@@ -0,0 +1,44 @@
+{
+  "app": {
+    "title": "Quiz Recap",
+    "practice": "Hier können Sie Ihr Wissen testen!",
+    "intro": "Nachdem Sie sich für die Größe eines Quiz entschieden haben bekommen Sie zufällige Fragen gestellt. Es gibt Fragen mit genau einer oder auch mehreren korrekten Antworten. Fragen werden sofort ausgewertet, sodass Sie gleich Feedback erhalten ob Sie richtig geantwortet haben. Am Ende des Quiz können Sie Videos zu bestimmten Fragen und Themen wiederholen bei denen es am meisten Verbesserungspotenzial gibt.",
```

This wording does not seem to align with the feature set.

```suggestion
    "intro": "Nachdem Sie sich für eine Größe des Quiz entschieden haben, erhalten Sie eine zufällige Auswahl an Fragen. Fragen mit einer richtigen Antwort werden durch einen Radio Button angezeigt. Fragen mit mehreren richtigen Antworten werden durch eine Checkbox angezeigt.",
```

Not sure about the translation of radio button or check box 🤔